File: db4oj/core/src/com/db4o/internal/handlers/array/MultidimensionalArrayHandler3.java
/* Copyright (C) 2008  Versant Inc.  http://www.db4o.com */

package com.db4o.internal.handlers.array;


/**
 * @exclude
 */
public class MultidimensionalArrayHandler3 extends MultidimensionalArrayHandler {
    
    protected ArrayVersionHelper createVersionHelper() {
        return new ArrayVersionHelper3();
    }

}

File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/Internal/Handlers/Array/MultidimensionalArrayHandler3.cs
/* Copyright (C) 2004 - 2008  db4objects Inc.  http://www.db4o.com */

using Db4objects.Db4o.Internal.Handlers.Array;

namespace Db4objects.Db4o.Internal.Handlers.Array
{
	/// <exclude></exclude>
	public class MultidimensionalArrayHandler3 : MultidimensionalArrayHandler
	{
		protected override ArrayVersionHelper CreateVersionHelper()
		{
			return new ArrayVersionHelper3();
		}
	}
}